util/benchmark/ec/p256_awslc_ossl/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#------------------------------------------------------------------------------------
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#------------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.15)
+project(benchmark_ec_p256 C)
+
+set(CMAKE_C_STANDARD 99)
+
+include_directories(.)
```

Comment:
Does this still work when using [`${CMAKE_CURRENT_SOURCE_DIR}`](https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_SOURCE_DIR.html) or [`${CMAKE_CURRENT_LIST_DIR}`](https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_LIST_DIR.html)?

If so, I think it makes sense to avoid using relative paths.

Reply:
Done: changed to ${CMAKE_CURRENT_SOURCE_DIR}. Thanks for the reference.